images/hook/entrypoint.sh

```diff
@@ -67,32 +67,21 @@ if [ $1 = "update" ]; then
         fi
     fi
 
-    echo "---> Creating new 'grafana' password"
-    password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -d '\n ' | /opt/bin/base64)
-    sed -i s/cGFzc3dvcmQtZ29lcy1oZXJlCg==/$password/g /var/lib/gravity/resources/grafana.yaml
-
     echo "---> Creating or updating resources"
-    for filename in security smtp influxdb grafana heapster kapacitor telegraf alerts
+    for name in security smtp grafana metrics-server alerts kube-state-metrics
+    do
+        rig upsert -f /var/lib/gravity/resources/${name}.yaml --debug
+    done
+
+    for file in /var/lib/gravity/resources/crds/*
     do
-        rig upsert -f /var/lib/gravity/resources/${filename}.yaml --debug
+        head -n -6 $file | /opt/bin/kubectl apply -f -
```

Comment:
What's "head -n -6" for?

Reply:
As you remember we talked about the problem that gravity parses crd and adds status fields at the end of the file, and all that field is empty, which leads to the error for kubectl apply. Just a temporary solution, because I am not sure how can we fix it from gravity side.

Reply:
Can you create a ticket that describes the problem pls?